tests/pyreverse/test_inspector.py

```diff
@@ -110,7 +110,7 @@ class Concrete23(Concrete1): pass
         ("Concrete0", ["MyIFace"]),
         ("Concrete1", ["MyIFace", "AnotherIFace"]),
         ("Concrete2", ["MyIFace", "AnotherIFace"]),
-        ("Concrete23", ["MyIFace", "AnotherIFace"]),
```

Comment:
The reason this test changed is that the unit test actually used the default value of `True` for the now removed `herited` parameter, but in production this parameter was always `False`. So after removing the parameter from the function signature I had to update this test case accordingly.